### DISCUSSION TITLE
Behavior of the DistanceUnits Input Tag

### DISCUSSION THREAD

**Discussion by prconlin** [OrderN/CONQUEST-release#164] [Ideas and suggestions]
The default distance unit is Bohr radii.  All input and output files use this unit unless otherwise specified.  The user can change this behavior by setting the tag
`General.DistanceUnits A `
With this setting enabled, Conquest will read coordinate files using Angstroms, and data written to Conquest_out will also use A.  

However, the coord_next.dat file is written in Bohr radii regardless of the DistanceUnits setting.  I am unsure if this behavior is intended.  In any case, I think it would be preferable to write the final coordinate file according to user settings.

**Comment by prconlin** [comment]
Also, the online documentation states that this tag is output only, but testing clearly shows that the tag also affects how the coords.dat file is read.  The documentation should be updated to reflect this behavior.

**Comment by davidbowler** [comment]
This is a good point, and we need to check this for all other output files (UpdatedAtoms.dat for instance).  I will create a more general issue to reflect this.
